upstream: https://github.com/docker/docker-py/pull/1313

+ /usr/bin/python3 setup.py build '--build-base=build-3'
Traceback (most recent call last):
  File "setup.py", line 36, in <module>
    long_description = readme_rst.read()
  File "/usr/lib64/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 152: ordinal not in range(128)
error: Bad exit status from /home/users/glen/tmp/rpm-tmp.71980 (%build)
    Bad exit status from /home/users/glen/tmp/rpm-tmp.71980 (%build)


--- docker-py-1.10.6/README.md~	2016-10-17 23:28:43.000000000 +0300
+++ docker-py-1.10.6/README.md	2016-11-27 14:52:18.175156781 +0200
@@ -3,7 +3,7 @@
 
 [![Build Status](https://travis-ci.org/docker/docker-py.png)](https://travis-ci.org/docker/docker-py)
 
-A Python library for the Docker Remote API. It does everything the `docker` command does, but from within Python – run containers, manage them, pull/push images, etc.
+A Python library for the Docker Remote API. It does everything the `docker` command does, but from within Python - run containers, manage them, pull/push images, etc.
 
 Installation
 ------------
--- docker-py-1.10.6/README.rst~	2016-11-03 01:49:01.000000000 +0200
+++ docker-py-1.10.6/README.rst	2016-11-27 14:53:00.603973652 +0200
@@ -4,7 +4,7 @@
 |Build Status|
 
 A Python library for the Docker Remote API. It does everything the
-``docker`` command does, but from within Python – run containers, manage
+``docker`` command does, but from within Python - run containers, manage
 them, pull/push images, etc.
 
 Installation
